examples/getstarted/src/admin/preview/dummy-preview.jsx

```diff
@@ -15,6 +15,20 @@ const PreviewComponent = () => {
     collectionType,
   });
 
+  React.useEffect(() => {
+    const handleStrapiUpdate = (event) => {
+      if (event.data?.type === 'strapiUpdate') {
+        window.location.reload();
+      }
```

Comment:
this works but it would be nice to demo the cache invalidation method here to avoid a slow hard refresh.

I think we could update useDocument to expose the refetch function and call it here